go.mod

```diff
@@ -69,7 +69,7 @@ require (
 	go.uber.org/atomic v1.11.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
 	golang.org/x/mod v0.22.0 // indirect
-	golang.org/x/net v0.38.0 // indirect
+	golang.org/x/net v0.33.0 // indirect
```

Comment:
Why do we need to downgrade this one?

Reply:
Surprisingly that since we upgraded it by bot: https://github.com/kubeflow/trainer/pull/2602

Reply:
thx